### DISCUSSION TITLE
标准 K8s 集群在线安装 - DaoCloud Enterprise

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#778] [Announcements]
# 标准 K8s 集群在线安装 - DaoCloud Enterprise

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/install/community/k8s/online/

**Comment by a-spit-to-silkworm-circle** [comment]
VERSION=v0.3.30，在线安装。
在线下载的安装器有问题，自己编译安装器可以用。
问题1：安装istio gateway那一步，istioVerson没有被引用，在线下载istio包时，url不正确。（因此自己编译安装器）

问题2：另外，安装完成后，通过VIP打不开dce5界面

**Reply by windsonsea** [reply]
@panpan0000 @jzhupup

**Reply by windsonsea** [reply]
> VERSION=v0.3.30，在线安装。 在线下载的安装器有问题，自己编译安装器可以用。 问题1：安装istio gateway那一步，istioVerson没有被引用，在线下载istio包时，url不正确。（因此自己编译安装器）
> 
> 问题2：另外，安装完成后，通过VIP打不开dce5界面

方便加一下社区版交流群吗，群里有很多开发人员，能够快速回答问题

https://docs.daocloud.io/install/intro/#_4

**Reply by a-spit-to-silkworm-circle** [reply]
v0.4.0的安装很顺利

**Comment by liyuerich** [comment]
通过 Helm 安装, Helm Chart 存放于 system repo 下:
离线环境下怎么拿到helm chart？

**Reply by liyuerich** [reply]
创建工作负载，以 MacVLAN 为例：- 这部分能放到容器管理中吗？集合网卡配置更清晰

**Reply by windsonsea** [reply]
这个评论好像错位了，不是这一页的内容吧：
https://docs.daocloud.io/install/community/k8s/online/
